### PR TITLE
add RC snippets

### PR DESCRIPTION
**Pull request by kroikie**
Add snippets for Firebase Remote Config.

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        console.error(err);
+      });
+}
+// [END validate_template]
+
+// [START add_new_condition]
+function addNewCondition(template) {
+  template.conditions.push({
+    name: 'android_en',
+    expression: 'device.os == \'android\' && device.country in [\'us\', \'uk\']',
```
Comment (samtstern) [inline config/index.js:22]: If you use `"` for the string you won't have to escape the `'`:

```
"device.ios == 'android' && device.country in ['us', 'uk']"
```

File: config/package.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "config",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```
Comment (samtstern) [inline config/package.json:6]: Check out the `package.json` for some of the other modules.  You should:

1. Make sure this is included in `lerna.json`
2. Get rid of the `test` script that failes
3. Add a `compile` script like the others

File: config/index.js (hunk 1 above)
Comment (kroikie) [reply config/index.js:22]: ESLint is yelling at me, should I ignore it?

File: config/package.json (hunk 2 above)
Comment (kroikie) [reply config/package.json:6]: Hmmm, this should have been in there. Thanks for pointing it out. I'll add it.

File: config/index.js (hunk 1 above)
Comment (samtstern) [reply config/index.js:22]: Maybe using ` instead of " will make it stop yelling?

File: config/package.json (hunk 2 above)
Comment (kroikie) [reply config/package.json:6]: It is still angry ` 22:17  error  Strings must use singlequote  quotes`

File: config/index.js (hunk 1 above)
Comment (kroikie) [reply config/index.js:22]: It is still angry 22:17 error Strings must use singlequote quotes

File: config/package.json (hunk 2 above)
Comment (kroikie) [reply config/package.json:6]: Done

File: config/index.js (hunk 1 above)
Comment (samtstern) [reply config/index.js:22]: Eh ok not worth fighting more, but thats strange

**Pull request merged**